Tests/ASDisplayNodeTests.mm

```diff
@@ -353,7 +353,7 @@ - (void)checkValuesMatchDefaults:(ASDisplayNode *)node isLayerBacked:(BOOL)isLay
   XCTAssertEqual((id)nil, node.accessibilityLabel, @"default accessibilityLabel is broken %@", hasLoadedView);
   XCTAssertEqual((id)nil, node.accessibilityHint, @"default accessibilityHint is broken %@", hasLoadedView);
   XCTAssertEqual((id)nil, node.accessibilityValue, @"default accessibilityValue is broken %@", hasLoadedView);
-  if (AS_AT_LEAST_IOS11) {
+  if (@available(iOS 11.0, *)) {
```

Comment:
Switching to `@available` to make CI happy.

Reply:
We're assuming everyone is using Xcode 9 now I guess?

Reply:
No, this is only for CI and contributors. Users aren't forced into Xcode 9.